pkg/linter/linter_test.go

```diff
@@ -727,8 +727,8 @@ import data.unresolved`,
 	}
 }
 
-// 1133583042 ns/op	3185226192 B/op	61392363 allocs/op
-// 1111341875 ns/op	3181965048 B/op	61390130 allocs/op
+// 1173608750 ns/op	3293593016 B/op	63366367 allocs/op
+// 1157190875 ns/op	3285082192 B/op	63014329 allocs/op
```

Comment:
What is that? 🔍

Reply:
You can use gobenchdata to block PRs  that spoil your performance too much, you know? 😉

Reply:
That would prevent us from adding new rules ;) Which is what these numbers reflect. New rules are "expensive" as even a simple one will scan 245 files or so..

Reply:
The recent improvements in OPA provides plenty of headroom for new rules to be added until we reach previous levels with less rules! But ultimately, I don't think we can continue to add, say 100 more rules without changing something more fundamental in the model, like "bundling" similar rules together to avoid each one starting from scratch, etc.... But a change like that most likely requires further improvements in OPA too. Like how we can't really use custom functions in hot paths (like `walk`) currently without a hefty penalty.